source/js/buyers-guide/components/filter/filter.scss

```diff
@@ -45,7 +41,7 @@
     justify-content: center;
     background: white;
 
-    @media screen and (min-width: 550px) {
+    @media (min-width: $bp-md) {
```

Comment:
I'm changing these to `$bp-md` so responsive design breakpoints can be consistent with mockups